itest/rust/src/register_tests/func_test.rs

```diff
@@ -89,6 +111,35 @@ impl GdSelfReference {
         f1 && f2
     }
 
+    #[func]
+    fn cfg_removes_function() -> bool {
+        true
+    }
+
+    #[cfg(any())]
+    #[func]
+    fn cfg_removes_function() -> bool {
+        false
+    }
+
+    #[func]
+    #[cfg(any())]
+    fn cfg_removes_function() -> bool {
+        false
+    }
```

Comment:
Could also use `compile_error!`, no?

This might also apply elsewhere...

Reply:
Missed it here. Thanks

Reply:
Added `compile_error!` to constants as well now.